web/webpack.config.js

```diff
@@ -59,17 +70,42 @@ const config = {
       },
       {
         test: /\.(woff2?|ttf|otf|eot|svg)$/,
-        loader: 'file-loader'
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: '[name].[ext]',
+              outputPath: 'static/fonts'
```

Comment:
This change is about moving fonts into a subfolder, just so there's less stuff at the top level.  No biggie.